airbyte_cdk/test/standard_tests/docker_base.py

```
@@ -0,0 +1,196 @@
+import sys
+import warnings
+from pathlib import Path
+
+import pytest
+import yaml
+from boltons.typeutils import classproperty
+
+from airbyte_cdk.models.connector_metadata import MetadataFile
+from airbyte_cdk.test.standard_tests.models import (
+    ConnectorTestScenario,
+)
```

Pylint (3.3.7)

[error] No name 'models' in module 'airbyte_cdk.test.standard_tests'

(E0611)